src/code42cli/cmds/cases.py

```diff
@@ -0,0 +1,246 @@
+from collections import OrderedDict
+
+import click
+from py42.exceptions import Py42BadRequestError
+from py42.exceptions import Py42NotFoundError
+
+from code42cli.click_ext.groups import OrderedGroup
+from code42cli.options import format_option
+from code42cli.options import sdk_options
+from code42cli.output_formats import OutputFormatter
+
+
+_HEADER_KEYS_MAP = OrderedDict()
```

Comment:
`OrderedDict` is no longer needed in the CLI, from the removal of py3.5 support.

Reply:
I've also starting favoring a helper method to create the dict rather than defining it at root level, but that is not a big deal

Reply:
Changed.